### PR TITLE
Mentality

### PR DESCRIPTION
**Pull request by KilnOfTheSecondFlame**

### CODE REVIEW

File: Thesis.tex [hunk 1]
```diff
@@ -169,6 +169,20 @@ \subsection{Closed Source Software}
...
+We can conclude that piracy was if not necessarily as widespread as it is today, it was prevalent enough, that voices for judicial protection never ceased to exist.
+
+The same was true for the creators of music in the 1900s, as sheet music was not deemed \textquoteleft useful arts\textquoteright\ and therefore not worthy of protection. Even after sheet music was being protected by the 1909 copyright act, which introduced the ingenious construct of 'compulsory licensing' (in which the creator can choose who produced the first recording of his music, and every other recorder just has to pay a flat fee), the recordings of said music was not itself protected. The justification of this disparity was that these recordings were merely parts of a machine and thus not a creation protected by copyright.
```
Comment (PixelSupreme) [inline Thesis.tex]: [...]the recordings of said music _were_ not itself protected.

File: Thesis.tex [hunk 2]
```diff
@@ -169,6 +169,20 @@ \subsection{Closed Source Software}
...
+
+In the same time, the efflorescence of radio and television led to the development of, what \cite{Lin2013} calls, the mentality of free. The customer is receiving content for \textquotedblleft free\textquotedblright\ from service providers, which in turn pay the licensing fee and use advertising to pay these and profit from their service. This business model was broadly adopted in the age of internet. The problem being, that the customer had to listen or view advertisements on radio or television, banner ads could be more easily ignored. During this time Peer-to-peer filesharing systems like Napster started to spring up, propelled by the ever increasing speeds of the domestic internet connection. And quoting from the excellent paper from \cite{Lunceford2008}: \textquotedblleft The logic of the consumer dictates that if a product is freely available on the market [through service providers], it is probably legal.\textquotedblright
```
Comment (PixelSupreme) [inline Thesis.tex]: propelled by the ever increasing speeds of domestic internet connections.

File: Thesis.tex [hunk 3]
```diff
@@ -169,6 +169,14 @@ \subsection{Closed Source Software}
 \section{Mentality concerning IP rights}
 \label{sec:IPMent}
 
+As mentioned in the section \ref{sec:Patents} the use of patents and copyright were to 'promote the Progress of science and useful arts'. This was adapted from the british system forged in 1710; under this system only books, maps, and charts were protected by copyright. \parencite[3]{Cummings2010}
```
Comment (NathDrac) [inline Thesis.tex]: \ref{sec:Patents},
"promote the Progress of science and useful arts".

File: Thesis.tex [hunk 4]
```diff
@@ -169,6 +169,14 @@ \subsection{Closed Source Software}
...
+
+These limitations were also applied to works produced in America. Interestingly enough, even works which were considered protected later on, were not eligible for copyright protection, if they were found to be of illicit nature.  Exactly this happened to the work \textit{Three Weeks} by Elinor Glynn, which was deemed so \textquotedblleft grossly immoral that copyright protection cannot exist\textquotedblright\ as is described in \cite{Fox1945}.
```
Comment (NathDrac) [inline Thesis.tex]: later on were not

File: Thesis.tex [hunk 5]
```diff
@@ -169,6 +169,14 @@ \subsection{Closed Source Software}
...
+We can conclude that piracy was if not necessarily as widespread as it is today, it was prevalent enough, that voices for judicial protection never ceased to exist.
```
Comment (NathDrac) [inline Thesis.tex]: was, if not necessarily as widespread as it is today, prevalent enough

File: bibliography.bib [hunk 6]
```diff
@@ -70,7 +70,7 @@ @misc{Rogers2017
 	year = {2017}
 }
 @misc{BSA2017,
-	abstract = {BSA is the leading advocate for the global software industry before governments and in the international marketplace. Its members are among the world's most innovative companies, creating software solutions that spark the economy and improve modern life.},
+	abstract = {BSA is the leading advocate for the global software industry before governments and in the international marketplace. Its members are among the world's most innovative companies, creating software solutions that spark th eeconomy and improve modern life.},
```
Comment (NathDrac) [inline bibliography.bib]: the economy

File: Thesis.tex (hunk 2 above)
Comment (NathDrac) [reply Thesis.tex]: The problem being that the customer had to listen or view advertisements on radio or television; banner ads, however, could be more easily ignored.

File: Thesis.tex [hunk 7]
```diff
@@ -174,8 +174,14 @@ \section{Mentality concerning IP rights}
...
+
+In the 2000s the RIAA was beginning to crack down on these so called file sharers. But in their scorched earth approach which not just targeted big copyright infringers, but also twelve year old kids, the music industry garnered the ire by the public. The public did not buy into their narrative that the music industry was the victim in this and the copyright deadlock which copyright owners now possessed (life of author plus 70 years) made it seem like just another cash grab by a powerful conglomerate. Another thing complicating the matter is people identifying themselves with the music, so it not longer belongs just to the creator and owner of copyright in the mind of the public. \parencite{Lunceford2008}
```
Comment (NathDrac) [inline Thesis.tex]: ire of the public

File: Thesis.tex [hunk 8]
```diff
@@ -174,8 +174,14 @@ \section{Mentality concerning IP rights}
...
+
+In the same time, the efflorescence of radio and television led to the development of, what \cite{Lin2013} calls, the mentality of free. The customer is receiving content for \textquotedblleft free\textquotedblright\ from service providers, which in turn pay the licensing fee and use advertising to pay these and profit from their service. This business model was broadly adopted in the age of internet. The problem being, that the customer had to listen or view advertisements on radio or television, banner ads could be more easily ignored. During this time Peer-to-peer filesharing systems like Napster started to spring up, propelled by the ever increasing speeds of the domestic internet connection. And quoting from the excellent paper from \cite{Lunceford2008}: \textquotedblleft The logic of the consumer dictates that if a product is freely available on the market [through service providers], it is probably legal.\textquotedblright
+
```
Comment (NathDrac) [inline Thesis.tex]: Note of the misleading use of the term piracy would not be too bad

File: Thesis.tex (hunk 5 above)
Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: We can conclude that piracy was, if not necessarily as widespread as it is today, prevalent enough that voices for judicial protection never ceased to exist.

**Review by PixelSupreme — APPROVED** [review]

**Pull request merged**